mumbleBot.py

```diff
@@ -653,6 +658,23 @@ def resume(self):
         self.wait_for_ready = True
         self.pause_at_id = ""
 
+    # map the volume_before_converting(0~1) into -60~0 dB, convert dB into a factor between 0~1.
+    def convert_set_volume(self, volume_before_converting):
+        self.unconverted_volume = volume_before_converting
+        dB = -60 + volume_before_converting * 60
+        self._volume_set = 10 ** (dB / 20)
```

Comment:
This again works, but I have a major issue with it which is that `0` maps to `10 ** (-C / 20)` (for `C` equal to `60`, `30` or whatever we choose). This means that the lowest volume is not silence, which can often be a problem - specially for ducking purposes.

What I would do here is simply substracting as a correction to the function this exact value:

```
self._volume_set = 10 ** (dB / 20) - 10 ** (-C / 20)
```

This would let `volume_before_converting` map to `0` whenever it itself becomes `0`, which I think is really important :)

Reply:
Sure.